python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -463,7 +463,7 @@ def location(self):
                     LFNs.append(this_url)
             return LFNs
 
-    def accessURL(self, thisSE=''):
+    def accessURL(self, thisSE='', protocol='xroot'):
```

Comment:
The default is determined within DIRAC so I'd rather not start requesting it here where we used to leave the default of `False`. I'm weary of moving away from the DIRAC default in-case anything changes.